lib/pdksync.rb

```diff
@@ -58,8 +58,17 @@ def self.main(steps: [:clone, :pdksync, :createpr], args: nil)
     @module_names = return_modules
     # The current directory is saved for cleanup purposes
     @main_path = Dir.pwd
-    exit 'No command passed' if args.nil? && (steps.first == :run_a_command)
-    puts "Running '#{args}'" if !args.nil? && (steps.first == :run_a_command)
+
+    # validation run_a_command
+    if steps.first == :run_a_command
+      raise 'No command passed' if args.nil?
+      puts "Command '#{args}'"
```

Comment:
I think you should change this from 'Command', as there technically wouldn't be any, to something along the lines of "PDKsync requires an argument to run".